dev-docs/bidders/rxnetwork.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: RxNetwork
+description: Prebid RxNetwork Bidder Adaptor
+pbjs: true
```

Comment:
You could set this and  `pbs_app_supported`  to false and we can get this merged.

You would then open a second pull request with the prebid-server PR linked and change it back to true. This way we are not blocked by the PBS PR review

Reply:
@muuki88 `pbs` and `pbs_app_supported` changed to `false`